pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -10133,6 +10133,18 @@ final: prev:
     meta.homepage = "https://github.com/vim-scripts/taglist.vim/";
   };
 
+  tailwindcss-colorizer-cmp-nvim = buildVimPlugin {
+    pname = "tailwindcss-colorizer-cmp.nvim";
+    version = "2023-09-09";
+    src = fetchFromGitHub {
+      owner = "roobert";
+      repo = "tailwindcss-colorizer-cmp.nvim";
+      rev = "bc25c56083939f274edcfe395c6ff7de23b67c50";
```

```suggestion
      rev = "3d3cd95e4a4135c250faf83dd5ed61b8e5502b86";
```
Latest version includes a bug fix from https://github.com/roobert/tailwindcss-colorizer-cmp.nvim/pull/13. Make sure to change the `version` and `sha256` as well.